spryker/nginx/etc/confd/templates/nginx/site_internal_requests.conf.tmpl

```diff
@@ -0,0 +1,3 @@
+if ($http_x_internal_request ~ 1) {
```

Comment:
how do we ensure this isn't set by anything external?

Reply:
Seems that setting `$config[ZedRequestConfig::IGNORE_SSL_CERT_SELF_SIGNING] = true;` allows us to talk to https.